SampleScripts/PowerManagement-WorkloadDomain.ps1

```diff
@@ -631,6 +667,9 @@ Try {
                 $runningAllVMs = Get-VMToClusterMapping -server $vcServer.fqdn -user $vcUser -pass $vcPass -cluster $cluster.name -folder "vm" -powerstate "poweredon" -silence
                 $runningVclsVMs = Get-VMToClusterMapping -server $vcServer.fqdn -user $vcUser -pass $vcPass -cluster $cluster.name -folder "vcls" -powerstate "poweredon" -silence
                 $runningVMs = $runningAllVMs | ? { $runningVclsVMs -notcontains $_ }
+                if($vxRailDetails -ne "") {
+                    $runningVMs = $runningAllVMs | ? { $vcfvms -notcontains $_ }
+                }
             }
             if ($runningVMs.count) {
                 Write-PowerManagementLogMessage -Type WARNING -Message "Some VMs are still in powered-on state." -Colour Cyan
```

Comment:
```suggestion
                Write-PowerManagementLogMessage -Type WARNING -Message "Some VMs are still in powered-on state." -Colour Yellow
```

Reply:
A `WARNING` should be `YELLOW`.

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108